crates/db/Cargo.toml

```diff
@@ -10,6 +10,7 @@ authors.workspace = true
 
 [dependencies]
 ethui-types.workspace = true
+ethui-networks.workspace = true
```

Comment:
most likely if you need to add this, we need to rethink how to structure the code. the reason we have all these separate crates is to avoid too many interdependencies, or circular dependencies. and having the db depend on the networks seems a bit off intuitively